build.sbt

```diff
@@ -328,32 +311,75 @@ lazy val rocketchip_inclusive_cache = (project in file("generators/rocket-chip-i
   .dependsOn(rocketchip)
   .settings(libraryDependencies ++= rocketLibDeps.value)
 
-// Library components of FireSim
-lazy val midas      = (project in file ("sims/firesim/sim/midas"))
-  .dependsOn(rocketchip, midasTargetUtils)
-  .settings(libraryDependencies ++= Seq(
-     "org.scalatestplus" %% "scalacheck-1-14" % "3.1.3.0" % "test"))
+lazy val fpga_shells = (project in file("./fpga/fpga-shells"))
+  .dependsOn(rocketchip, rocketchip_blocks)
+  .settings(libraryDependencies ++= rocketLibDeps.value)
+  .settings(commonSettings)
+
+lazy val chipyard_fpga = (project in file("./fpga"))
+  .dependsOn(chipyard, fpga_shells)
+  .settings(commonSettings)
+
+// Components of FireSim
+
+lazy val firesimAsLibrary = sys.env.get("FIRESIM_STANDALONE") == None
+lazy val firesimDir = if(firesimAsLibrary) {
+  file("sims/firesim")
+} else {
+  file("sims/firesim-staging/firesim-symlink")
+}
```

Comment:
TODO: Switch to not using an env. var but always properly check the existence of a file from the Chipyard symlink.

Reply:
Leaving this for later.